docs/cloud-provider-credentials/obtain-cloud-providers-credentails.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Get Cloud Provider Credentials
```

Comment:
File name says `obtain...` and here it says `get...`, I'd suggest to stick to `get` and standardize the usage of the same verb for similar operations.

Reply:
file removed, now using the [get-gcp-service-account-credentials.md](https://github.com/devopness/devopness/blob/614e3070a24c883a787e519c4821679fe2bbfc67/docs/cloud-provider-credentials/get-gcp-service-account-credentials.md) file